src/_guides/language/effective-dart/design.md

```diff
@@ -743,6 +743,24 @@ Future<bool> install(PackageId id, String destination) {
 
 With types, all of this is clarified.
 
+
+### DON'T specify that a setter returns `void`.
+
+Although the `void` type is technically correct, it's pointless noise.
```

Comment:
Saying it's "technically correct" might lead readers to believe it adds some information, but it really doesn't. The language infers `void` for setters anyway, so it adds nothing. Maybe just say:

> The type system infers `void` for all setters automatically.